src/VisualStudio/Setup.Dependencies/ProvideRoslynBindingRedirection.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using Microsoft.VisualStudio.Shell;
+
+namespace Roslyn.VisualStudio.Setup
+{
+    /// <summary>
+    /// A <see cref="RegistrationAttribute"/> that provides binding redirects with all of the Roslyn settings we need.
+    /// It's just a wrapper for <see cref="ProvideBindingRedirectionAttribute"/> that sets all the defaults rather than duplicating them.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Assembly, AllowMultiple = true)]
+    internal sealed class ProvideRoslynBindingRedirectionAttribute : RegistrationAttribute
```

Comment:
Oh, is this a copy/paste of the other type but we didn't rename the type? Let's pick a different name. The ProvideRoslyn* name is intended for Roslyn binaries as it encodes some versioning decisions. If these version differently, a different name should be used.